### PR TITLE
[GS End User][W-8528117] Create Label and add the End user Resources list

### PR DESCRIPTION
**Pull request by fparodi-salesforce**
# Critical Changes

# Changes
Create a new list of Resources for End Users
# Issues Closed

# Community Ideas Delivered

# New Metadata
- Custom Label
  - gseuResourcesItemAboutNPSPLabel
  - gseuResourcesItemAboutNPSPLinkLabel
  - gseuResourcesItemCommunityGroupLabel
  - gseuResourcesItemCommunityGroupLinkLabel
  - gseuResourcesItemQuestionsAnsweredLabel
  - gseuResourcesItemQuestionsAnsweredLinkLabel
  - gseuResourcesItemWatchVideosLabel
   - gseuResourcesItemWatchVideosLinkLabel
   - gseuResourcesItemWebinarsLabel
   - gseuResourcesItemWebinarsLinkLabel

# Deleted Metadata

### CODE REVIEW

**Comment by fparodi-salesforce** [comment]
[W-8528117](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008kdcFIAQ/view)

**Review by force2b — APPROVED** [review]

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -13910,4 +13910,84 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
         <shortDescription>Video header title in Get Started</shortDescription>
         <value>Get Started on NPSP</value>
     </labels>
+    <labels>
+        <fullName>gseuResourcesItemQuestionsAnsweredLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource Questions Answered label in GS Resources section for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:13918]: Please change to: Questions Answered label in Get Started Resources section for End Users.

File: src/labels/CustomLabels.labels [hunk 2]
```diff
@@ -13910,4 +13910,84 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
...
+        <value>Get Your Questions Answered</value>
+    </labels>
+    <labels>
+        <fullName>gseuResourcesItemQuestionsAnsweredLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource Questions Answered link label in GS Resources section for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:13926]: Please change to: Questions Answered link label in Get Started Resources section for End Users.

File: src/labels/CustomLabels.labels [hunk 3]
```diff
@@ -13910,4 +13910,84 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
...
+        <value>Power of Us Hub</value>
+    </labels>
+    <labels>
+        <fullName>gseuResourcesItemAboutNPSPLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource About NPSP label in GS Resources section for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:13934]: Please change to: About NPSP label in Get Started Resources section for End Users.

File: src/labels/CustomLabels.labels [hunk 4]
```diff
@@ -13910,4 +13910,84 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
...
+        <value>Learn More About NPSP</value>
+    </labels>
+    <labels>
+        <fullName>gseuResourcesItemAboutNPSPLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource About NPSP link label in GS Resources section for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:13942]: Please change to: About NPSP link label in Get Started Resources section for End Users.

File: src/labels/CustomLabels.labels [hunk 5]
```diff
@@ -13910,4 +13910,84 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
...
+        <value>NPSP Documentation</value>
+    </labels>
+    <labels>
+        <fullName>gseuResourcesItemWatchVideosLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource Watch Videos label in GS Resources section for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:13950]: Please change to: Watch Videos label in Get Started Resources section for End Users.

File: src/labels/CustomLabels.labels [hunk 6]
```diff
@@ -13910,4 +13910,84 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
...
+        <value>Watch How-To Videos</value>
+    </labels>
+    <labels>
+        <fullName>gseuResourcesItemWatchVideosLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource Watch Videos link label in GS Resources section for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:13958]: Please change to: Watch Videos link label in Get Started Resources section for End Users.

File: src/labels/CustomLabels.labels [hunk 7]
```diff
@@ -13910,4 +13910,84 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
...
+        <value>NPSP Videography Channel</value>
+    </labels>
+    <labels>
+        <fullName>gseuResourcesItemWebinarsLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource Webinars label in GS Resources section for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:13966]: Please change to: Webinars label in Get Started Resources section for End Users.

File: src/labels/CustomLabels.labels [hunk 8]
```diff
@@ -13910,4 +13910,84 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
...
+        <value>Find Events and Webinar</value>
+    </labels>
+    <labels>
+        <fullName>gseuResourcesItemWebinarsLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource Webinars link label in GS Resources section for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:13974]: Please change to: Webinars link label in Get Started Resources section for End Users.

File: src/labels/CustomLabels.labels [hunk 9]
```diff
@@ -13910,4 +13910,84 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
...
+        <value>Salesforce Calendar</value>
+    </labels>
+    <labels>
+        <fullName>gseuResourcesItemCommunityGroupLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource Community Group link label in GS Resources section for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:13982]: Please change to: Community Group label in Get Started Resources section for End Users.

File: src/labels/CustomLabels.labels [hunk 10]
```diff
@@ -13910,4 +13910,84 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
...
+        <value>Find a Community Group</value>
+    </labels>
+     <labels>
+        <fullName>gseuResourcesItemCommunityGroupLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource Community Group link label in GS Resources section for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:13990]: Please change to: Community Group link label in Get Started Resources section for End Users.

**Review by smarovich — APPROVED** [review]
Labels look good. Thanks @fparodi-salesforce!

**Pull request merged**